examples/utility/SelfProvisioning/SelfProvisioning.ino

```diff
@@ -0,0 +1,493 @@
+/*
+  Self Provisioning on Arduino IoT Cloud
+
+  This sketch demonstrates how to automate the provisioning
+  of the Arduino boards MKR WiFi 1010 and Nano 33 IoT to
+  the Arduino IoT Cloud, using the Arduino Cloud API.
+
+  Circuit:
+   - MKR WiFi 1010 board
+   - Nano 33 IoT board
+
+  created 14 August 2020
+  by Luigi Gubello
+
+  This example code is in the public domain.
+*/
+
+#include "arduino_secrets.h"
+#include "ECCX08TLSConfig.h"
+#include <ArduinoBearSSL.h>
```

Comment:
`ArduinoBearSSL` has been integrated into `ArduinoIoTCloud:master:head` so there is no need to include this here, sketch compiles fine without it.

Reply:
> ArduinoBearSSL has been integrated into ArduinoIoTCloud:master:head so there is no need to include this here, sketch compiles fine without it.

 :+1: :100: 

> when testing it on MKR WiFi 1010 and PROD I was unable to connect to the server

Are you using Arduino IoT Cloud 0.10 and NINA-fw 1.3.0? Can you change `delay(5000);` in `delay(10000);` and try again? Thank you!

Reply:
Please first add this to CI build, that alone usually catches all kinds of problems.